registry/cloudflare-workers/index.js

```diff
@@ -29,12 +34,17 @@ const removeRoute = async ({ zone_id }, context) => {
     'X-Auth-Email': `${EMAIL}`,
     'X-Auth-Key': `${AUTH_KEY}`
   }
-
   const response = await fetch(url, { headers, method: `DELETE` })
+
+  if (response.status == 200) {
+    context.log(`✅ Route disbaled: ${response.statusText}`)
```

Comment:
nit: `disbaled` should be `Disabled`, also if we are capitalizing every word in other logs, this should be capitalized as well